stock_move_actual_date/models/stock_picking.py

```diff
@@ -14,3 +14,9 @@ def _get_actual_date_update_triggers(self):
     def _get_stock_moves(self):
         self.ensure_one()
         return self.move_ids
+
+    def _create_backorder(self):
+        backorders = super()._create_backorder()
+        moves = backorders.mapped("move_ids")
+        moves.write({"actual_date_source": False})
```

Comment:
Just to be slightly more efficient.
```suggestion
        backorders.move_ids.filtered("actual_date_source").actual_date_source = False
```

Reply:
Done